string input_string
---
int32 output_int